dependencies.gradle

```diff
@@ -84,6 +84,13 @@ ext {
       // google-api-client major version 1.
       'com.google.apis:google-api-services-dns:[v1-rev20240419-2.0.0, v2beta)',
 
+      // Later versions of jcommander have breaking changes that we do not wish to pick up
```

Comment:
I'm curious what the breaking change is, and how do we subsequently resolve this block?

Reply:
it's in the 2.x versions, it parses sets of command line arguments differently -- I didn't do enough digging to figure out what would be necessary to fix it